packages/dds/matrix/src/test/matrix.stress.spec.ts

```diff
@@ -301,9 +308,13 @@ describe("Matrix", () => {
                         /* seed: */ (Math.random() * 0x100000000) >>> 0,
                     );
 
-                    // console.log(matrices[0].toString());
-                    // eslint-disable-next-line max-len
-                    console.log(`Stress loop: ${++iterations} iterations completed - Total Elapsed: ${((Date.now() - start) / 1000).toFixed(2)}s`);
+                    // Note: Mocha reporter intercepts 'console.log()' so use 'process.stdout.write' instead.
+                    process.stdout.write(matrices[0].toString());
+
+                    process.stdout.write(
+                        `Stress loop: ${++iterations} iterations completed - Total Elapsed: ${
+                            ((Date.now() - start) / 1000).toFixed(2)
+                        }s\n`);
```

Comment:
^-- The above console output only appears in the "long haul" stress.  (i.e., not during 'npm t', in the lab, etc.)